megatron/arguments.py

```diff
@@ -678,6 +679,9 @@ def _add_network_size_args(parser):
                        help='Untie embeddings and output weights.'),
     group.add_argument('--embedding-weights-in-fp32', action='store_true',
                        help='Cast word embedding weights to fp32 before embedding fwd.'),
+    group.add_argument('--kill-switch-path', type=str, default=None,
+                       help='Path to look for a kill switch. '
```

Comment:
@polisettyvarma, is the kill switch meant to be a file or a folder, or both? Can this be clarified in the help message?